src/www/ui/api/documentation/openapi.yaml

```
@@ -1628,6 +1628,46 @@ paths:
                 $ref: '#/components/schemas/Info'
         default:
             $ref: '#/components/responses/defaultResponse'
+  /license/import:
```

I would prefer adding CSV to the endpoint name signifying import file type.
```suggestion
  /license/import-csv:
```

---

done